go/client/cmd_compat.go

```diff
@@ -10,26 +10,6 @@ import (
 
 // compatibility with node client commands:
 
-func NewCmdCompatEncrypt(cl *libcmdline.CommandLine) cli.Command {
```

Comment:
Sounds good. I'll punt on this for now, looks like  https://keybase.atlassian.net/browse/CORE-2070 is already tracking this.